packages/server/src/tests/banners/ChannelBannerTests.ts

```diff
@@ -68,7 +72,19 @@ const BannerVariantFromParams = (forChannel: BannerChannel) => {
             }
         };
 
-        const tickerSettings = undefined;
+        const tickerSettings =
+            variant.template === BannerTemplate.UsEoyMomentBanner
+                ? {
+                      countType: TickerCountType.money,
+                      endType: TickerEndType.unlimited,
+                      currencySymbol: '$',
+                      copy: {
+                          countLabel: 'contributions',
+                          goalReachedPrimary: "We've hit our goal!",
+                          goalReachedSecondary: 'but you can still support us',
+                      },
+                  }
+                : undefined;
```

Comment:
We'll need to make sure if this is the copy that we want. The `goalReached` parts aren't super important just yet as it'll be a while until that's live - but we need to make sure the `countLabel` is correct!

Reply:
Yeah I think a typo had been made here, we want the `countLabel` to say contributed